examples/inspiral/run.sh

```diff
@@ -1,16 +1,15 @@
-#!/bin/bash -e
```

Comment:
I think you've reverted a bunch of my changes to this test. It's important that we make sure this runs for different FFT threads and backends. Please incorporate changes into this if needed, but not revert (I suspect this wasn't intentional, but if it was we should discuss).

Reply:
Yes, sorry, I didn't mean to commit this actually. It was just for testing beforehand. I'll revert this back.